app/models/billing_expense.rb

```diff
@@ -14,4 +14,8 @@ class BillingExpense < ApplicationRecord
   def self.amount_collection
     AMOUNT.map(&:to_s)
   end
+
+  def billing_hours
+    Hour.where(billing_expense: id).count
```

Comment:
`hours.size` should do the same

(`#size` on ActiveRecord collections only runs the query once, vs. `#count` which runs it everytime)